src/main/steps/applicant1/review-the-application/content.ts

```diff
@@ -144,7 +131,7 @@ const en = ({ isDivorce, userCase, partner, userEmail, isApplicant2 }: CommonCon
     userCase.applicant2SolicitorFirmName ? userCase.applicant2SolicitorFirmName : 'not given'
   }`,
   solAddress: `Solicitor address: ${
-    userCase.applicant2SolicitorAddress ? userCase.applicant2SolicitorAddress : 'not given'
+    userCase.applicant2SolicitorAddress?.trim() ? userCase.applicant2SolicitorAddress : 'not given'
```

Comment:
does this _userCase.applicant2SolicitorAddress_ not need to be joined with the other fields like below? 

i.e 
       [
          userCase.applicant2SolicitorAddress1,
          userCase.applicant2SolicitorAddress2,
          userCase.applicant2SolicitorAddress3,
          userCase.applicant2SolicitorAddressTown,
          userCase.applicant2SolicitorAddressCounty,
          userCase.applicant2SolicitorAddressPostcode,
          userCase.applicant2SolicitorAddressCountry,
        ]
      .filter(Boolean)
      .join('\<br>\');

Reply:
if you mean data wise, then no. applicant2SolicitorAddress has the info, the FE solicitor address is multiple input boxes but the backend/xui is a string/textarea so when we post the info, its converted to a string with new line e.g. test\npostcode\ncountry and stored in applicant2SolicitorAddress.
if you mean format wise, James is away and there hasnt been an indication on how it should look, i did show it to Michael and he was okay with it but i think he is looking at it from a content wise. I did change it for the repondent address format as i think it should be formated with new lines.